src/test/java/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,53 @@
+package service;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String PATH_TO_FILE = "src/test/resources/testValidData.csv";
+    private static final String FILE_NAME = "testValidData.csv";
+    private static List<String> expected;
+
+    @BeforeClass
+    public static void operationBeforeTest() {
+        expected = new ArrayList<>();
+    }
+
```

Comment:
You can join logic of  lineWithFile() method to @BeforeClass method  You don't need both of them. This way before starting tests you'll have ready List to work with.